File: Semestru 3/Rețele de calculatoare/Laboratoare/Lab3/Exemple/c1.php
 <?php
 /* Enunt: Clientul trimite serverului un numar. Serverul il primeste si il afiseaza pe ecran.
	
 Rulare in doua terminale diferite:
	php server.php
	php client.php
 */ 
 $s = socket_create(AF_INET, SOCK_STREAM, 0);
 socket_connect($s , "127.0.0.1",4321);
 $nr=readline("Dati un numar: \n");
 socket_send($s, $nr, 100, 0);
 socket_recv($s,$nr,100,0);
 echo "Am primit de la server $nr\n";
 socket_close($s);
?>

File: Semestru 3/Rețele de calculatoare/Laboratoare/Lab3/Exemple/c1.py
#Enunt: Clientul trimite serverului un numar. Serverul il primeste si il afiseaza pe ecran.
#
#Rulare in doua terminale diferite:
#	python server.py
#	python client.py
import socket

TCP_IP = "127.0.0.1"
TCP_PORT = 4321
MESSAGE = str(input("Dati un numar: "))

s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
s.connect((TCP_IP, TCP_PORT))
s.send(MESSAGE)
print "Am trimis la server: ",MESSAGE
data = s.recv(10)
print "Am primit de la server: ",data
s.close()
